src/data/super_gameboy/commands.asm

```diff
@@ -0,0 +1,156 @@
+;
+; Super GameBoy pre-defined commands
+;
+
+; Defines a Super Game Boy command.
+; Usage:
+;   sgb_cmd <command code>, <length>
+sgb_cmd: macro
+    db   (\1) << 3 + (\2)
+endm
+
+; Defines a Super Game Boy 'Data Send' command.
+; Usage:
+;   sgb_data_snd <SNES destination address>, <SNES destination bank>, <data length>
+sgb_data_send_cmd: macro
+    sgb_cmd SGB_DATA_SND, 1
+    dw  \1 ; SNES destination address
+    db  \2 ; SNES destination bank
+    db  \3 ; data length
+endm
+
+; Displays a black screen, regardless of the VRAM content.
+SGBSetScreenMaskBlackCmd::
+    sgb_cmd SGB_MASK_EN, 1
+    db   2  ; black-screen mask
+    ds   14 ; padding
+
+; Make the Super Game Boy display the GB screen again.
+SGBCancelMaskCmd::
+    sgb_cmd SGB_MASK_EN, 1
+    db   0  ; cancel mask
+    ds   14 ; padding
+
+SGBInit1Cmd::
+    sgb_data_send_cmd $085D, $00, $0B
+.data
+    db   $8C, $D0, $F4, $60, $00, $00, $00, $00, $00, $00, $00
```

Comment:
Might be nicer to copy the format used here: https://github.com/pret/pokered/blob/master/data/sgb_packets.asm#L228

Also see:
 - https://tcrf.net/Notes:Super_Game_Boy#DATA_SND
 - https://forums.nesdev.com/viewtopic.php?f=12&t=16610#p206526